### DISCUSSION TITLE
Unsupported HTTP method "get" for this action.

### DISCUSSION THREAD

**Discussion by Sujaz** [gurock/trcli#243] [General]
Once we upgraded out Testrail to V8.0.1, trcli fails to publish results.

Command:
trcli -y --project '_Modular BIGIP (MBIP)' -h https://testrail.olympus.f5net.com/testrail -u [MASKED] -k [MASKED] -v --insecure parse_junit --title 'DUMMY' --suite-id 116699 --run-id 179532 -f dep_test.xml

 trcli isprinting the following error:
"response status code: 404
response body: {'error': 'Unsupported HTTP method "get" for this action.'}"

Is it something new or local authorization issue?

**Comment by jakec-routable** [comment]
Are you sure you have the right host? Does your host url really include the path `/testrail`? If the host is correct, then look a little further up in the output log for the request that is returning the `Unsupported HTTP method`. When you have found it, try making the same request via curl to see if you can successfully curl the same request.

**Reply by Sujaz** [reply]
That resolved the issue!
removed the /testrail and now it works fine. Thank you!
